CONTRIBUTING.md

```diff
@@ -19,6 +19,9 @@ To setup a local environment, perform the following steps:
 - Install dependencies: `npm install`
 - Run the build `npm run build` (only required once to copy assets)
 
+> [!NOTE]
+> If you modify `/src/helper/config.schema.json`, run `npm run prepare` to regenerate the dependent files.
```

Comment:
Should also indicate that this needs done when first getting started on a fresh repo too

Reply:
The `prepare` command is a special command in NPM that is run when `npm i` is run, so there's no need to specifically run it on fist setup!